src/main/java/de/qabel/desktop/ui/transfer/TransactionFxProgressCollectionModel.java

```diff
@@ -0,0 +1,37 @@
+package de.qabel.desktop.ui.transfer;
+
+import de.qabel.desktop.daemon.management.HasProgressCollection;
+import de.qabel.desktop.daemon.management.Transaction;
+import javafx.beans.property.BooleanProperty;
+import javafx.beans.property.ObjectProperty;
+import javafx.beans.property.SimpleBooleanProperty;
+import javafx.beans.property.SimpleObjectProperty;
+
+public class TransactionFxProgressCollectionModel extends FxProgressCollectionModel<Transaction> {
+	private ObjectProperty<Transaction> currentTransaction = new SimpleObjectProperty<>();
+	private BooleanProperty running = new SimpleBooleanProperty(false);
+
+	public TransactionFxProgressCollectionModel(HasProgressCollection<?, Transaction> progress) {
+		super(progress);
+		onChange(this::update);
+	}
+
+	private void update(Transaction transaction) {
+		if (transaction == null || transaction.getState() != Transaction.STATE.RUNNING) {
```

Comment:
between two transfers (when the last one is finished and the new one hasn't been started yet)
extract method? ...